.github/workflows/gradle-release.yml

```
@@ -0,0 +1,50 @@
+        git tag ${{ env.VERSION_NAME }}
+        git push origin ${{ env.VERSION_NAME }}
+      env:
+        GITHUB_TOKEN: ${{ github.token }}
+
+    - name: Upload APK to Release
+      uses: softprops/action-gh-release@v2
+      with:
+        tag_name: '${{ env.VERSION_NAME }}'
+        files: ./app/build/outputs/apk/release/${{ env.APP_ID }}-${{ env.VERSION_CODE }}-release.apk
+      env:
+        GITHUB_TOKEN: ${{ github.token }}
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)